frontend/src/components/map/setups/setup-terra-draw.ts

```diff
@@ -50,17 +50,19 @@ export const setupTerraDraw = (map: maplibregl.Map) => {
           if (updateType === "finish" || updateType === "commit") {
             return ValidateNotSelfIntersecting(feature);
           }
-          return true;
```

Comment:
Hi @JamesLMilner , thank you for this PR and for the great library! I've tested everything, and it works fine.
However, there seems to be a typescript error here (see attached). Could you suppress this please ?

<img width="881" alt="Screenshot 2025-03-22 at 09 28 11" src="https://github.com/user-attachments/assets/9ed01482-1c65-4b13-84cc-f1d2a05948ae" />

Reply:
I checked the library, and `Validation` comes from this line, which means it's correct. But for some reason, typescript is frowning at it.

https://github.com/JamesLMilner/terra-draw/blob/eedef9f59e368a42fbdecb9a1106110d0956b760/packages/terra-draw/src/common.ts#L105

Reply:
Actually, nevermind, the code builds now @JamesLMilner . Sorry for the confusion! Thank you! 

@kshitijrajsharma , kindly proceed to merge this.

Reply:
Deal !